docs/internals.rst

```diff
@@ -0,0 +1,332 @@
+.. _internals:
+
+Internals
+=========
+
+Dredd itself is a `command-line <https://en.wikipedia.org/wiki/Command-line_interface>`__ `Node.js <https://nodejs.org/>`__ application written in modern JavaScript.
+
+
+.. contents::
+   :local:
+   :depth: 1
+
+
+Maintainers
+-----------
+
+`Apiary <https://apiary.io/>`__ is the main author and maintainer of Dredd’s `upstream repository <https://github.com/apiaryio/dredd>`__. Currently responsible people are:
+
+
+-  `@paraskakis <https://github.com/paraskakis>`__ - product decisions, feature requests
+-  `@honzajavorek <https://github.com/honzajavorek>`__ - lead of development
+
+Dredd supports so many programming languages only thanks to the work of several contributors, who were so awesome they dedicated time to create, improve, and maintain the respective :ref:`hook handlers <hooks>`:
+
+-  `@ddelnano <https://github.com/ddelnano>`__ (:ref:`PHP <hooks-php>`, :ref:`Go <hooks-go>`)
+-  `@gonzalo-bulnes <https://github.com/gonzalo-bulnes>`__ (:ref:`Ruby <hooks-ruby>`)
+-  `@hobofan <https://github.com/hobofan>`__ (:ref:`Rust <hooks-rust>`)
+-  `@snikch <https://github.com/snikch>`__ (:ref:`Go <hooks-go>`)
+-  `@ungrim97 <https://github.com/ungrim97>`__ (:ref:`Perl <hooks-perl>`)
+
+
+.. _contributing:
+
+Contributing
+------------
+
+We are grateful for any contributions made by the community. Even seemingly small contributions such as fixing a typo in the documentation or reporting a bug are very appreciated!
+
+To learn the basics of contributing to Dredd, please read the `contributing documentation <https://github.com/apiaryio/dredd/blob/master/CONTRIBUTING.md#readme>`__, placed in Dredd's GitHub repository.
+
+
+Installing Dredd for development
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To hack Dredd locally, clone the repository and run ``npm install`` to install JavaScript dependencies. Then run ``npm test`` to verify everything works as expected.
```

Comment:
Perhaps a note that you can run the dev-Dredd via `./bin/dredd` would be nice, I'd expect if someone is contributing then they will likely want to test it directly out.